### PR TITLE
Do not colorize if printing to non-TTY device

### LINKED ISSUE #791: [Feature Request] 支持输出不带ANSI转义序列的纯文本日志
如下图，如果想在Qt应用中显示运行日志的话，使用QProcess读到的运行日志是带有ANSI转义符的，输出到文本框后就会看起来一团糟。
同样，将运行日志输出至文件（例如`unblockneteasemusic-win-x64.exe > log.txt`）时，输出的日志仍然带有ANSI转义符。

是否能像GCC一样，在输出到非TTY设备时，输出未经ANSI转义的纯文本？

详见：[Color output in console - ArchWiki](https://wiki.archlinux.org/title/Color_output_in_console#Reading_from_stdin)
> When you run a command and pipe its [standard output](https://en.wikipedia.org/wiki/Standard_output) (_stdout_) to less for a paged view (e.g. `pacman -Qe | less`), you may find that the output is no longer colored. This is usually because the program tries to detect if its stdout is an interactive terminal, in which case it prints colored text, and otherwise prints uncolored text. This is good behaviour when you want to redirect stdout to a file, e.g. `pacman -Qe > pkglst-backup.txt`, but less suited when you want to view output in `less`.
test_NpcqiZUiL5

### PR DESCRIPTION
**Pull request by FrzMtrsprt**
Resolves #791, hopefully :)

### CODE REVIEW

File: src/logger.js [hunk 1]
```diff
@@ -3,17 +3,19 @@ const pino = require('pino');
...
 	{
 		level: process.env.LOG_LEVEL ?? 'info',
 		prettyPrint:
 			process.env.JSON_LOG === 'true'
 				? false
 				: {
-						colorize: true,
-						messageFormat:
-							'\x1b[1m\x1b[32m({scope})\x1b[0m\x1b[36m {msg}',
-						ignore: 'time,pid,hostname,scope',
```
Comment (1715173329) [inline src/logger.js:16]: Hmmm, why did you remove this?

Comment (FrzMtrsprt) [reply src/logger.js:16]: It was an accident...

**Review by 1715173329 — APPROVED** [review]
LGTM.

**Review by pan93412 — APPROVED** [review]
